src/pages/global.tsx

```diff
@@ -33,24 +68,195 @@ export default function Page() {
   };
 
   return (
-    <VStack spacing={componentSpacing} width={maxTextWidth} align="start">
-      <PageBreadcrumb current="全局配置" />
-
+    <>
       <FormControl>
         <FormLabel>初次交流反馈表截止时间，格式: 2022-02-02T01:01:01+08:00</FormLabel>
         <Input
           defaultValue={data?.matchFeedbackEditableUntil}
           onChange={(e) => setMatchFeedbackEditableUntil(e.target.value)}
         />
       </FormControl>
-
       <Button
         variant="brand"
         onClick={save}
         isLoading={saving}
       >
         保存
       </Button>
-    </VStack>
+    </>
+  );
+}
+
+function MeetingSlots() {
+  const meetingSlotQuery = trpcNext.meetings.listMeetingSlots.useQuery();
+  const meetingSlots = meetingSlotQuery.data as MeetingSlot[] | undefined;
+
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const [editingSlot, setEditingSlot] = useState<MeetingSlot | null>(null);
+  const [formValues, setFormValues] = useState({
+    tmUserId: '',
+    meetingId: '',
+    meetingLink: ''
+  });
+  const [isSaving, setIsSaving] = useState(false);
+
+  const createOrUpdateMutation = trpcNext.meetings.createOrUpdateMeetingSlot.useMutation({
+    onSuccess: () => {
+      const action = editingSlot ? '更新' : '创建';
+      toast.success(`会议位置${action}成功`);
+      void meetingSlotQuery.refetch();
+      handleCloseModal();
+    },
+    onError: (error) => {
+      const action = editingSlot ? '更新' : '创建';
+      toast.error(`${action}失败: ${error.message}`);
+    },
+    onSettled: () => setIsSaving(false)
+  });
+
+  const handleCreateNew = () => {
+    setEditingSlot(null);
+    setFormValues({ tmUserId: '', meetingId: '', meetingLink: '' });
+    onOpen();
+  };
+
+  const handleEdit = (slot: MeetingSlot) => {
+    setEditingSlot(slot);
+    setFormValues({
+      tmUserId: slot.tmUserId || '',
+      meetingId: slot.meetingId,
+      meetingLink: slot.meetingLink
+    });
+    onOpen();
+  };
+
+  const handleCloseModal = () => {
+    onClose();
+    setEditingSlot(null);
+    setFormValues({ tmUserId: '', meetingId: '', meetingLink: '' });
+  };
+
+  const handleSave = async () => {
+    setIsSaving(true);
+    const payload = {
+      tmUserId: formValues.tmUserId,
+      meetingId: formValues.meetingId,
+      meetingLink: formValues.meetingLink,
+      // Only include id if we're editing an existing slot
+      ...(editingSlot && { id: editingSlot.id }),
+    };
+    await createOrUpdateMutation.mutateAsync(payload);
```

Comment:
don't use async. don't create the createOrUpdateMutation variable. don't catch errors. just do

```
try {
  trpc.meetings.createOrUpdateMeetingSlot.mutate...;
  toast.success();
} finally {
  setIsSaving(false);
}
```